modules/oadp-operator-supported.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-intro.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oadp-operator-supported_{context}"]
+= {oadp-full} support
+
+.Supported versions of {oadp-short}
+[width="100%",cols="20%,20%,20%,20%,20%, options="header",]
+|===
+
+|Version
+|{OCP-short} version
+|General availability
+|Full support ends
+|Maintenance ends
+
+|1.4
+|4.14, 4.15, 4.16
+|10 Jul 2024
+|Release of 1.5
+|Release of 1.6
+
+|1.3
+|4.12, 4.13, 4.14, 4.15
+|29 Nov 2023
+|10 Jul 2024
+|Release of 1.5
```

Comment:
line 29 here will need to change in our official support statement as well.  I need to update that too, sorry.
line 29 should read as close to https://access.redhat.com/support/policy/updates/openshift for OCP 4.14 as possible.

* Extended update support ends:  Oct 31, 2025
* Extended update support Term 2 ends: October 31, 2026

Reply:
extended update support Term1 and Term 2 must be on OCP 4.16

Reply:
extended update support Term1 and Term 2 must be on OCP 4.14